### ISSUE TITLE
Where did Recent Discussions go

### ISSUE THREAD

**Issue by callmekatootie** [topcoder-platform/forums#426]
Earlier, when I would visit https://discussions.topcoder.com/ - recent discussions would show up. This was a good way for me to track any open threads that I needed to respond and not rely on email notifications.

But now, I see the UI has changed - I am seeing Roundtables instead. Further, I see a compact version - there is no information about when a comment was last added or who added.

**Comment by sdgun** [comment]
@callmekatootie I know this might be a useful feature to some of us, but this was intentional feature change that went in today's release.

**Comment by callmekatootie** [comment]
Thanks @sdgun ... do you happen to know the relevant ticket that brought this change...

**Comment by sdgun** [comment]
@callmekatootie  https://github.com/topcoder-platform/forums/issues/376

**Comment by jmgasper** [comment]
@Oanh-and-only-Oanh ☝️

**Comment by Oanh-and-only-Oanh** [comment]
@jmgasper how does "Recent Discussions" diff from Bookmarking or getting notifications for activity on a thread you've been active on?

**Comment by jmgasper** [comment]
@Oanh-and-only-Oanh - Recent Discussions is just a list things that have been updated recently on anything you have access to.  This includes roundtables and challenge forums.

Bookmarking is just specific discussions you care about, and notifications are only sent on forums you "watch".

**Comment by jmgasper** [comment]
We have "Bookmarks", like this:  <img width="292" alt="Screen Shot 2021-02-23 at 11 20 17 am" src="https://user-images.githubusercontent.com/21790/108787082-1e2bdf00-75c9-11eb-8dc9-4c4f9aa60b2a.png">

What if we made "Recent Discussions" that only showed **unread** threads that you haven't viewed yet?  With a counter of unread as well, like the `1` in the screenshot above?  Would you want that to include roundtables?

**Comment by Oanh-and-only-Oanh** [comment]
Sure. I like the unread option. Just want to be aware of our timeline and don't want release anything extra that could affect our release/timeline.

Question is either this version or the next... what's your effort and timeline estimate

**Comment by jmgasper** [comment]
@Oanh-and-only-Oanh - This would be next release, I think.  I don't want to mess with this for the current release, where we have less than a week left.  I'd rather focus on the current performance updates.

**Comment by Oanh-and-only-Oanh** [comment]
agreed. let's add this to the next version then. Nick and I will have a full list for you soon.
